src/main/java/org/junit/internal/runners/JUnit38ClassRunner.java

```diff
@@ -170,6 +174,13 @@ public void sort(Sorter sorter) {
         }
     }
 
+    public void order(GenericOrdering ordering) throws InvalidOrderingException {
```

Comment:
(Not editing earlier comment in case you're currently responding, but "Why the duplication?" is too strong.  More to the point: "I see the case of the target type of OrderWith.  Are there other places?")